packages/flutter_tools/lib/src/migrations/lldb_init_migration.dart

```diff
@@ -0,0 +1,146 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:xml/xml.dart';
+import 'package:xml/xpath.dart';
+
+import '../base/file_system.dart';
+import '../base/logger.dart';
+import '../base/project_migrator.dart';
+import '../build_info.dart';
+import '../ios/xcodeproj.dart';
+import '../project.dart';
+
+class LLDBInitMigration extends ProjectMigrator {
+  LLDBInitMigration(IosProject project, BuildInfo buildInfo, {required Logger logger})
```

Comment:
Can you use `super.logger` here?

```suggestion
  LLDBInitMigration(IosProject project, BuildInfo buildInfo, super.logger)
```

Prior art: https://github.com/flutter/flutter/blob/9246b02af12ce64cb8b1a3339d4c42c2324cabc9/packages/flutter_tools/lib/src/ios/migrations/remove_bitcode_migration.dart#L11